src/NotifyQueueConsumer/Command/Handler/SendToNotifyHandler.php

```diff
@@ -46,9 +48,20 @@ public function handle(SendToNotify $sendToNotifyCommand): UpdateDocumentStatus
             throw new UnexpectedValueException("Cannot read PDF");
         }
 
+
         // 3. Send to notify
-        list('id' => $notifyId, 'status' => $notifyStatus)
-            = $this->sendToNotify($sendToNotifyCommand->getUuid(), $contents);
+        if ($sendToNotifyCommand->getSendBy() === 'email') {
+            $response = $this->sendInvoiceToNotify(
```

Comment:
I'm late to the party, but I find this a bit awkward: if anything is sent from Sirius with `sendBy=email`, it'll be treated like an invoice and use the invoice template.

I think it'd be clearer to either have the message specify that it's an invoice (e.g. `sendBy=invoice`) or specify the template to use (e.g. `sendBy=email,template=daef7d83-9874-4dd8-ac60-d92646e7aaaa`).